superset/utils/event_logger.py

```diff
@@ -22,17 +22,77 @@
 import json
 import logging
 import textwrap
-from typing import Any, cast, Type
+import time
+from typing import Any, cast, List, Type
 
-from flask import current_app, g, request
+from flask import current_app, g, request, Response
+
+SUPERSET_EVENT_LOGGER_NAME = "superset_events"
+
+
+class SupersetEvent(object):
```

Comment:
NIT: deriving `object` is an "out-of-fashion" py2 best practice, not needed anymore in py3

Reply:
heh I'm not even sure why I did that, I think some dormant pathway in my brain woke up.  I've got a lot of fixes coming, I'll remove that.